### ISSUE TITLE
Object Detection API recognise 1 class with small prediction

### ISSUE THREAD

**Issue by programowalny** [tensorflow/models#5189]
### System information
What is the top-level directory of the model you are using: object_detection
Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Window 10
TensorFlow installed from (source or binary): conda installation
TensorFlow version (use command below): 1.8
Bazel version (if compiling from source): None
CUDA/cuDNN version: 9.0
GPU model and memory: GeForce GTX 1050TI (Laptop)
Exact command to reproduce: None

### Describe the problem
 Hello, i Have problem with traininf any model on my dataset to predict, some classes. 

Firstly, i strated configure object detection api , fallowing by this tutorial:
https://github.com/EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10

I used 7 thousands images in 25 categories(road signs), I used rcnn architectura or ssd, all fails.  After this i limit my categories to 5 and images to 2 thousands, i create accurate bounding boxes and i check it twice, and I eliminate a mistakes.  I run model_main.py and even a legacy/train.py. On the second output i saw a strage values in loss function. I radomly get a very high values.  Please give me some advise, where can i find mistakes, I fallow step by step by tutorail , taht i mentioned, so it can't a configuration mistake

**Comment by harshini-gadige** [comment]
This question is better asked on [StackOverflow](https://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.
